File: FirstBit Workspace/C Programming/Assignments/Assignment 7/func with no return type and with para/Assignment 0/Q8/Q8.c
//Write a C program to input the length and width of a rectangle and find its perimeter. (p = 2 * (length + width))

#include<stdio.h>

void rectanglePerimeter(float*, float*, float*);
void main(){
	
	//initialization
	float length = 4, width = 5.6;
	float perimeterOfRectangle;
	
	rectanglePerimeter(&length, &width, &perimeterOfRectangle);
	
	//printing output
	printf("perimeter of rectangle is %.2f",perimeterOfRectangle);
	
}

void rectanglePerimeter(float* length, float* width, float* perimeterOfRectangle){

	//performing operation
	*perimeterOfRectangle = 2 * (*length + *width);
	
}

File: FirstBit Workspace/C Programming/Assignments/java_to_c_connectivity/Assignment 0/Q8/Q8.java
//Write a C program to input the length and width of a rectangle and find its perimeter. (p = 2 * (length + width))


class Q8{
	public static void main(String[] args){
		
		//initialization
		double length = 4, width = 5.6;
		
		//declaration
		double perimeterOfRectangle;
		
		//performing operation
		perimeterOfRectangle = 2 * (length + width);
		
		//printing output
		System.out.printf("perimeter of rectangle is %.2f",perimeterOfRectangle);
		
	}
}


